plugins/BEdita/API/config/routes.php

```diff
@@ -148,7 +148,15 @@ function (RouteBuilder $routes) {
             [
                 '_namePrefix' => 'model:',
             ],
-            $resourcesRoutes($modelingControllers)
+            function ($routes) use ($modelingControllers, $resourcesRoutes) {
```

Comment:
If possible I'd add a type hinting to function's parameter, as it helps IDEs: `function (RouteBuilder $routes) ...`